content/en/docs/Badges/MCH2022/software-development/rp2040/_index.md

```diff
@@ -5,33 +5,35 @@ nodateline: true
 weight: 1
 ---
 
+## Introduction
 
-# This voids your warranty!
-# SERIOUSLY WE WON'T SUPPORT YOU IF YOU BRICK YOUR BADGE
+RP2040: aka Raspberry Pico. This is an onboard coprocessor that handles
```

Comment:
This was added at request of Renze ... did you talk to him about it?

While I agree with you, I also share the concern, that people will start playing around with it and we'll have to support them to repair the Badge.

Reply:
I think as a hacker it is good to know that every state is resolvable without soldering JTAG to tiny pins or something. Renze stated in the chat that he is sure that recover is possible. I am not sure on his opinion whether to encourage people to experiment there. He provided me with the new link to the UF2 file, but did not comment otherwise. We could add notes on cloning, building and flashing the ESP32 firmware with make prepare make as this all what is required.

Reply:
Added note "not recommended".

Reply:
By the way, is there a simple flash tool to manually re-flash an ESP32 binary taken from the Over-the-Air update site?